test/serialbox-python/run_python_tests.sh.cmake

```diff
@@ -17,12 +17,12 @@
 #
 # Export paths
 #
-export PYTHONPATH=$PYTHONPATH:${SERIALOBX_PYTHON_MODULE}:${SERIALOBX_PYTHON_MODULE}/sdb
+export PYTHONPATH="$PYTHONPATH:${SERIALOBX_PYTHON_MODULE}:${SERIALOBX_PYTHON_MODULE}/sdb"
```

Comment:
Just saw a typo in the SERIALOBX_PYTHON_MODULE var

Reply:
Nice catch! `sdb` is abandoned anyway though ;)